src/main/java/seedu/address/model/consultation/Time.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.model.consultation;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+/**
+ * Represents a Time in the system.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTime(String)}
+ */
+public class Time {
+
+    public static final String MESSAGE_CONSTRAINTS = "Times should be in the format HH:MM, "
```

Comment:
Did you mean "Times should be in the format HH:mm"? If I'm not mistaken, MM is Month & mm is Minute.